pkg/api/dashboard_librarypanels.go

```diff
@@ -0,0 +1,60 @@
+package api
+
+import (
+	"errors"
+
+	"github.com/grafana/grafana/pkg/components/simplejson"
+	"github.com/grafana/grafana/pkg/models"
+	"github.com/grafana/grafana/pkg/services/librarypanels"
+)
+
+func (hs *HTTPServer) loadLibraryPanels(dash *models.Dashboard) error {
```

Comment:
How about moving the `loadLibraryPanels` method to `LibraryPanelService` instead (and call it LoadDashLibraryPanels for clarity)? It would make HTTPServer less fat, and I don't see any reason not to do it.

Reply:
Good suggestion, I'll change that.

Reply:
Fixed!